taskt/Core/Script/ScriptAction.cs

```diff
@@ -9,13 +8,13 @@ public class ScriptAction
         /// <summary>
         /// generic 'top-level' user-defined script command (ex. not nested)
         /// </summary>
-        [XmlElement(Order = 1)]
```

Comment:
Does Json not have an order attribute? I'm curious

Reply:
It does, but json as a standard is unordered so I thought it would be best to not force any specific ordering.